shadows/framework/src/main/java/org/robolectric/shadows/ShadowNotification.java

```diff
@@ -148,4 +148,18 @@ private View findView(View view, String resourceName) {
     }
     return subView;
   }
+
+  public boolean isAutoCancel() {
+    return (realNotification.flags & Notification.FLAG_AUTO_CANCEL)
+        == Notification.FLAG_AUTO_CANCEL;
+  }
+
+  public boolean isLocalOnly() {
```

Comment:
It might needs a RequiredApi, because `FLAG_LOCAL_ONLY` is added from SDK 20. See https://developer.android.com/reference/android/app/Notification#FLAG_LOCAL_ONLY.

Reply:
Thanks for the review, i have updated

Reply:
The `isLocalOnly` might also need `RequiresApi`.